dominion-solutions/n8n
# Ethan sowell/cu camhjr/build a clockify writer for projects

# Details

Added ClockifyWriter with support for creating projects.

# Steps to test changes

1. Use **npm run dev** to start the localhost.
1. Add a Clockify regular node.
1. Select "Create" and "Project" and put in all the information required, then click execute.
1. Make sure information returned is right and that a project is made.
> Returns existing project info if your project name is a duplicate.

# Issues Addressed

[CU-camhjr](https://app.clickup.com/t/camhjr) - Build a Clockify Writer for Projects
